README.md

```diff
@@ -155,6 +152,30 @@ auto-generated. E.g. `bundle.yaml` originates from the _Jsonnet_ files in
 2. Run `make generate-in-docker`.
 3. Commit the generated changes.
 
+## How to add / remove Go dependencies
+
+This project uses [go module](https://github.com/golang/go/wiki/Modules).
+
+We do "vendoring" (committing dependencies code into project).
+
+Once you upgrade `go.mod` or via `go get` command, make sure to do `go mod vendor` to download 
```

Comment:
Nits :) WDYT of this:

>After updating the dependencies in the `go.mod`, which can also be done by using the `go get github.com/example/repo`, execute the `go mod vendor`, do update the `vendor` directory. Once all the files are updated, commit the changes made to `go.mod`, `go.sum` and the `vendor` directory.

Should we mention running `go tidy` as well?

Reply:
Thanks, I improved this a bit based on your suggestions - tidy is must have as well (: